### DISCUSSION TITLE
Eighths

### DISCUSSION THREAD

**Discussion by NapalmCodes** [rxhanson/Rectangle#623] [Ideas]
Would it be possible to get a layout supporting eigths.
I want to achieve:

1/8    -- 2/8  --  1/8
1/8    -- 2/8  --  1/8

**Comment by rxhanson** [comment]
Unfortunately not with Rectangle, but it can be achieved with the custom shortcut builder in my paid superset of Rectangle, Hookshot (https://hookshot.app)
